java-runtime-builder-app/src/test/resources/testWorkspaces/customBuildWorkspace/app.yaml

```diff
@@ -0,0 +1,4 @@
+env: flex
+runtime: java
+runtime_config:
+  build_script: "bazel build //:all"
```

Comment:
This doesn't actually work - this is just a unit test fixture intended to test the build_script setting (and this build_script isn't actually executed). Perhaps not the best example anyway though, I'll update to a more realistic example.